jupyter/nmm-cache.ipynb

```diff
@@ -140,12 +146,45 @@
     "del read"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "1e0a1e14",
+   "metadata": {},
+   "source": [
+    "Die Methode `clean()` prüft ob der Cache seine maximale Größe überschritten hat. Ist dies der Fall wird das minimale Limit um eins erhöht und alle Einträge, deren Limit geringer ist, werden aus dem Cache entfernt."
```

Comment:
FYI: Wir schreiben in der Doku meist `Rekursionslimit` statt `limit`, damit man genau weiß was für ein Limit gemeint ist.

Reply:
fixed